From 3106d16bbc45de52bc0a8b638a84aa9d7c9abc46 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Wed, 6 Oct 2021 17:43:16 +0200
Subject: [PATCH] configure.ac: fix autoreconf with autoconf >= 2.70

Fix the following build failure raised with openocd and autoconf >= 2.70
due to
http://git.savannah.gnu.org/gitweb/?p=autoconf.git;a=commit;h=aba75f6d4a9c875a9d5d90a07c6b3678db66a4bf:

>>> openocd 0.11.0 Autoreconfiguring
autoreconf: error: configure.ac: AC_INIT not found; not an autoconf script?

Fixes:
 - http://autobuild.buildroot.org/results/5fb7aa28703aff61ba850eac11bd35c8804528ae

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://github.com/msteveb/jimtcl/pull/209]
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/jimtcl/configure.ac b/jimtcl/configure.ac
index b70b5d8..8c45114 100644
--- a/jimtcl/configure.ac
+++ b/jimtcl/configure.ac
@@ -1 +1,3 @@
 # Dummy configure.ac to make automake happy
+AC_INIT([jimtcl], [0.80])
+AC_OUTPUT
-- 
2.33.0

